src/common/socket.c

```diff
@@ -238,7 +238,11 @@ fd_set readfds;
 
 // Maximum packet size in bytes, which the client is able to handle.
 // Larger packets cause a buffer overflow and stack corruption.
-static size_t socket_max_client_packet = 24576;
+#if PACKETVER >= 20131223
+static size_t socket_max_client_packet = 0x10000;
```

Comment:
I just got someone who pointed me to that commit, and what's the issue, So I searched up in hercules forum, and found out your post saying about max packet size..